platform-sdk/swirlds-common/src/main/java/com/swirlds/common/NodeId.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.swirlds.common.system;
+package com.swirlds.common;
```

Comment:
Why moving it to the top package? In general I would suggest moving it to platform in future. so maybe `com.swirlds.common.platform`?

Reply:
This couldn't be moved in this PR due to dependencies on it from metrics and a few other things. Will put it into `com.swirlds.common.platform` in the short term.